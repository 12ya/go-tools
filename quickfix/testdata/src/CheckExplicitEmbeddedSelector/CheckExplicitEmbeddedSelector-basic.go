package pkg

type BasicOuter struct{ BasicInner }
type BasicInner struct{ F1 int }

func fnBasic() {
	var basic BasicOuter
	_ = basic.BasicInner.F1 // want `could remove anonymous field "BasicInner" from selector`
	_ = basic.F1            // minimal form
}
