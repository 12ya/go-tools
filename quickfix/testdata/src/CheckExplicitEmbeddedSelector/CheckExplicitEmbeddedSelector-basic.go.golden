package pkg

type BasicOuter struct{ BasicInner }
type BasicInner struct{ F1 int }

func fnBasic() {
	var basic BasicOuter
	_ = basic.F1 // want `could remove anonymous field "BasicInner" from selector`
	_ = basic.F1 // minimal form
}
